app/mailers/revision_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class RevisionMailer < ActionMailer::Base
+  TO_ADDRESS = 'welsh.editors@moneyadviceservice.org.uk'
+  SUBJECT    = 'Content updated by External Editor'
+
+  default from: ENV['MAILJET_DEFAULT_FROM_ADDRESS']
```

Comment:
I don't know what the convention around environment variables is in the project, but I learned a handy tip recently that helps identify errors. If you use `ENV.fetch('key')` it will raise an error saying the key doesn't exist if you forget to set it.

It's slightly longer than the above, which will just return `nil` if the key doesn't exist, but it helps when you're trying to debug why it isn't working and it's because you forgot to set an environment variable.